assets/js/googlesitekit/modules/datastore/modules.test.js

```diff
@@ -802,49 +803,28 @@ describe( 'core/modules modules', () => {
 
 		describe( 'getModuleFeatures', () => {
 			it( 'returns undefined when no modules are loaded', async () => {
-				fetchMock.getOnce(
-					/^\/google-site-kit\/v1\/core\/modules\/data\/list/,
-					{ body: FIXTURES, status: 200 }
-				);
-				const slug = 'analytics';
-				const featuresLoaded = registry.select( STORE_NAME ).getModuleFeatures( slug );
+				muteFetch( /^\/google-site-kit\/v1\/core\/modules\/data\/list/, [] );
+				const featuresLoaded = registry.select( STORE_NAME ).getModuleFeatures( 'analytics' );
 
 				// The modules will be undefined whilst loading.
 				expect( featuresLoaded ).toBeUndefined();
 			} );
 
 			it( `returns features when modules are loaded`, async () => {
-				fetchMock.getOnce(
-					/^\/google-site-kit\/v1\/core\/modules\/data\/list/,
-					{ body: FIXTURES, status: 200 }
-				);
-				const slug = 'analytics';
-				registry.select( STORE_NAME ).getModuleFeatures( slug );
-
-				// Wait for loading to complete.
-				await untilResolved( registry, STORE_NAME ).getModules();
+				registry.dispatch( STORE_NAME ).receiveGetModules( FIXTURES );
 
-				const featuresLoaded = registry.select( STORE_NAME ).getModuleFeatures( slug );
+				const featuresLoaded = registry.select( STORE_NAME ).getModuleFeatures( 'analytics' );
 
-				expect( fetchMock ).toHaveFetchedTimes( 1 );
-				expect( featuresLoaded ).toMatchObject( fixturesKeyValue[ slug ].features );
+				// expect( fetchMock ).toHaveFetchedTimes( 1 );
+				expect( featuresLoaded ).toMatchObject( fixturesKeyValue.analytics.features );
 			} );
 
-			it( `returns an empty array when requesting features for a non-existent module`, async () => {
-				fetchMock.getOnce(
-					/^\/google-site-kit\/v1\/core\/modules\/data\/list/,
-					{ body: FIXTURES, status: 200 }
-				);
-				const slug = 'non-existent-slug';
-				registry.select( STORE_NAME ).getModuleFeatures( slug );
-
-				// Wait for loading to complete.
-				await untilResolved( registry, STORE_NAME ).getModules();
+			it( `returns an empty object when requesting features for a non-existent module`, async () => {
```

Comment:
Ah, fair point. My guess is it was copied from one that does? There's a rule for it (`quotes`) that will prefer single quotes but allow for doubles to avoid escaping and backticks for template literals only. I guess we just don't have it enabled.

Filed #3056. 🙂